valhalla/skadi/sample.h

```diff
@@ -2,23 +2,31 @@
 #define __VALHALLA_SAMPLE_H__
 
 #include <cstdint>
+#include <functional>
+#include <future>
 #include <memory>
+#include <mutex>
 #include <string>
 #include <unordered_map>
+#include <unordered_set>
```

Comment:
nitpick: i think most of these changes can move to the cc file

Reply:
Done